.github/workflows/test.yml

```diff
@@ -109,3 +109,18 @@ jobs:
       - run: docker run --interactive --volume "$PWD:/schema" jsonschema
       - run: ./test/docker/help.sh
       - run: ./test/docker/bundle_fmt.sh
+
+  snap:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build snap
+        id: snapcraft
+        uses: snapcore/action-build@v1
+        with:
+          path: .
+      - name: Upload snap artefact
+        uses: actions/upload-artifact@v4
+        with:
+          name: snap-${{ github.sha }}.snap
+          path: ${{ steps.snapcraft.outputs.snap }}
```

Comment:
Can you also manually install the `.snap` file and run i.e. `jsonschema version` just to confirm it runs?

Reply:
Let me try this myself in the mean-time. I'll push a commit